src/20_Components/amp-social-share.html

```diff
@@ -0,0 +1,87 @@
+<!--
+  #### Introduction
+
+  The Social Share extension provides a common interface for share buttons, styled to complement each other.
+-->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <!-- #### Setup -->
+  <!--
+    Ensure you've configured your canonical URL.
+
+    Import the `amp-social-share` component in the header.
```

Comment:
can you move `Import the `amp-social-share` component in the header.` into it's own comment block above the import statement. This makes sure the sections are properly aligned.